### PR TITLE
Bugfixing

### PR DESCRIPTION
**Pull request by elozel**
Solved problem with:
name of variables and database columns, date, spamming buttons, empty fields, css, duplicate events

### CODE REVIEW

File: app/Http/Controllers/SaveDataController.php [hunk 1]
```diff
@@ -23,33 +23,53 @@ public function edit($id)
...
     
 
     public function SaveData(Request $request){
+        $taskDate = $request->input('taskDate');
+        $taskName = $request->input('taskName');
+        $taskDesc = $request->input('taskDesc');
+        $category = $request->input('category');
+        $priority = $request->input('priority');
+
+         $existingtask = taskStorage::where('taskDate', $taskDate)
```
Comment (Vvieczi) [inline app/Http/Controllers/SaveDataController.php:38]: wyrównanie wcięć

File: app/Http/Controllers/SaveDataController.php [hunk 2]
```diff
@@ -23,33 +23,53 @@ public function edit($id)
...
+                                  ->where('taskDesc', $taskDesc)
+                                  ->where('category', $category)
+                                  ->where('priority', $priority)
+                                  ->exists();
+
+       
+        if ($existingtask) {
+            return response()->json(['Spotkanie o takich danych już istnieje.']);
+        }
+
```
Comment (Vvieczi) [inline app/Http/Controllers/SaveDataController.php:49]: usuwanie pustych lini

File: public/css/edit.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+body {
```
Comment (Vvieczi) [inline public/css/edit.css:1]: html , ids , classes

File: public/css/main.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+body{
```
Comment (Vvieczi) [inline public/css/main.css:1]: html, ids, classes

File: resources/views/index.blade.php [hunk 5]
```diff
@@ -80,5 +71,30 @@
      </div>
 </div>
 <script src="/bootstrap/js/bootstrap.js"></script>  {{-- Added bootstrap js link --}}
+<script>
+    function submitForm() {
+        if(document.getElementById("taskDate").value!="" && document.getElementById("taskName").value!="" && document.getElementById("taskDesc").value!=""){
+            if(document.getElementById("taskDate").value-new Date("Y-m-d")>=0){
```
Comment (Vvieczi) [inline resources/views/index.blade.php:77]: separate to js file

**Review by Vvieczi — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/SaveDataController.php (hunk 1 above)
Comment (elozel) [reply app/Http/Controllers/SaveDataController.php]: done

File: app/Http/Controllers/SaveDataController.php (hunk 2 above)
Comment (elozel) [reply app/Http/Controllers/SaveDataController.php:48]: done

File: public/css/edit.css (hunk 3 above)
Comment (elozel) [reply public/css/edit.css:1]: done

File: resources/views/index.blade.php (hunk 5 above)
Comment (elozel) [reply resources/views/index.blade.php]: done

**Pull request merged**